includes/shareable-content.php

```diff
@@ -98,6 +98,7 @@
  * @var HTML $license_statement
  */
 $license_statement = wp_kses_post( get_option( 'republication_tracker_tool_policy' ) );
+$license_key = get_option( 'republication_tracker_tool_license' );
```

Comment:
Same as in `class-widget.php` - we should probably have a default value to avoid errors in accessing the array.